app/lib/allowed_cross_namespace_usage.rb

```diff
@@ -7,4 +7,5 @@
 module AllowedCrossNamespaceUsage
   VendorAPIApplicationPresenter = VendorAPI::ApplicationPresenter
   RegisterAPISingleApplicationPresenter = RegisterAPI::SingleApplicationPresenter
+  VendorAPIVERSION = VendorAPI::VERSION
```

Comment:
```suggestion
  VendorAPIVersion = VendorAPI::VERSION
```

Reply:
Keeping this upcase makes it easier to identify that this represents a constant, which is why we used this format.